preview5/GitHub.StartPage.Preview5.csproj

```diff
@@ -228,6 +228,10 @@
       <Project>{9aea02db-02b5-409c-b0ca-115d05331a6b}</Project>
       <Name>GitHub.Exports</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\..\src\GitHub.UI\GitHub.UI.csproj">
```

Comment:
Simply adding this reference won't do anything (except perhaps cause a CopyLocal behavior to copy the DLL to your output dir, if it wasn't already). If you inspect the resulting .dll you're building, I think you'll find the reference evaporated during the compilation because you weren't actually using it.
Since you're doing this to try to fix the image load problem, the only way this could work is if you actually _used_ the DLL at runtime so that it loaded. And that would just hurt perf since you're loading more than you naturally need. I'll provide a recommendation for a better fix on #655.